src/gdash/scripts/_contains/_contains.gml

```diff
@@ -15,9 +15,10 @@ _contains("hello", "ello");
 // => true
 */
 
-var collection = argument[0];
-var target = argument[1]
-var fromIndex = 0;
+var 

```

Comment:
As for `_push` and `_concat`, I guess it does make sense to mutate the input array for `_push`, given that push generally has that connotation. As for `_concat`, I'm less thrilled about mutating there, though it may just be my JS heritage, since in JS, `Array.concat` doesn't mutate the input, it returns a new array.

Perhaps its misguided, I'll look into similar gml functions and see if I can find anything inspiring.